lib/rspec/abq/version.rb

```diff
@@ -1,6 +1,6 @@
 module RSpec
   module Abq
     # current version!
-    VERSION = "1.1.1"
```

Comment:
this is actually a breaking change. Should we observe semver and bump to `2.0.0`? We've been avoiding it because there have been a number of breaking changes. Also we don't really observe semver anywhere yet.

... but if not semver, what do the three numbers represent? kind of whim. I'd call this a minor [breaking] change because it shouldn't change anything for anybody else.